api4/api.go

```diff
@@ -59,6 +59,8 @@ type Routes struct {
 	Plugins *mux.Router // 'api/v4/plugins'
 	Plugin  *mux.Router // 'api/v4/plugins/{plugin_id:[A-Za-z0-9\\_\\-\\.]+}'
 
+	PluginIntegrations *mux.Router // 'api/v4/pluginIntegrations'
```

Comment:
I prefer to avoid clashing urls. since below `api/v4/plugins/` anything can be a plugin id, there might be a plugin with id `integrations`.
Regarding naming, I am 100% flexible. I just want to keep in mind that the idea of this is not just have actions, but potentially full views. That is the reason I thought about "integrations".

Reply:
I still like to come up with a name that doesn't include an existing concept like `plugins`. Plugins are tied to the rpc layer, which this whole idea is not. It's HTTP only.

Because of that, I'm strongly think a new route like `api/v4/triggers` or `api/v4/actions` should be used.

Reply:
We should converge on the name here before we merge this given that it would be a breaking change to the API if we want to update to something else down the line.